src/Controller/PartyController.php

```diff
@@ -116,13 +116,14 @@ public function edit(CaseEntity $case, Party $party, CasePartyRelationRepository
     {
         $this->denyAccessUnlessGranted('edit', $case);
 
+        /** @var CasePartyRelation $relation */
+        $relation = $relationRepository->findOneBy(['case' => $case, 'party' => $party]);
+
         $form = $this->createForm(PartyFormType::class, null, [
             'case' => $case,
+            'type' => $relation->getType(),
```

Comment:
Are we sure that `$relation` cannot be `null` here?

Reply:
I should never happen, but i've added a check for it.